src/app/icon/icon.component.html

```diff
@@ -42,6 +42,18 @@
 		<img *ngIf=" iconName == 'white-shield' " src="../../assets/apps-icon.svg" alt="apps"
 			[style.width]=" iconSize + 'px' " [style.height]=" iconSize + 'px' ">
 
+		<!-- check -->
+		<svg *ngIf=" iconName == 'check'" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+			<path fill="var(--clear)" 
```

Comment:
Please use currentColor if we want to reuse this.
```suggestion
			<path fill="currentColor" 
```
Then set `color: var(--clear)` in `app-icon`.